sklearn/pipeline.py

```diff
@@ -1606,7 +1623,17 @@ def fit_transform(self, X, y=None, **fit_params):
             Targets for supervised learning.
 
         **fit_params : dict, default=None
```

Comment:
this needs to be renamed to `params` now that it can also be routed to `transform`.